tests/operator/operator.go

```diff
@@ -3367,3 +3305,67 @@ func patchKvCertConfig(virtClient kubecli.KubevirtClient, name string, certConfi
 	_, err = virtClient.KubeVirt(flags.KubeVirtInstallNamespace).Patch(context.Background(), name, types.JSONPatchType, data, metav1.PatchOptions{})
 	return err
 }
+
+func patchOperator(virtClient kubecli.KubevirtClient, newImageName, version *string) bool {
```

Comment:
Personally, I prefer it like that. It removes dependency in the singleton, if we'll want to change it someday. But have no strong feeling about it.

Reply:
I hear you. Singletons are generally a problem because they create this "star" dependency. However, since so many functions across the `tests` directory needs it, passing it around clutters their API in a painful manner. I have a strong feeling about adding it now to these helper functions.